code/modules/vehicles/multitile/multitile_verbs.dm

```diff
@@ -327,12 +327,87 @@
 	if(honk_sound)
 		playsound(loc, honk_sound, 75, TRUE, 15)	//heard within ~15 tiles
 
+//Finally, a broadcaster!
+/obj/vehicle/multitile/proc/use_broadcaster()
+	set name = "Use Broadcaster"
+	set desc = "Allows to send a loud message to the exterior via hull broadcaster."
+	set category = "Vehicle.General"
```

Comment:
/proc and it's using set? doesn't check out (also this would be better as an action button anyway)

Reply:
Well, that's how vehicle controls work.